policy.md

```diff
@@ -196,6 +134,45 @@ Where splitter must be one of the following:
 * \-
 * \~
 
+### 5.2 Auditor Qualifications ###
+
+CAs must submit a summary of the Audit Team's qualifications and experience as outlined below with respect to the audit. The information can also be provided as part of the audit result documentation, like the [ETSI Audit Attestation Letter (AAL)](https://www.acab-c.com/downloads/), or as a supplement to the WebTrust Assurance Report.
+
+* Date that the audit report was signed,
+* Full name of the CA that was audited,
+* Name and address of the audit firm or Conformity Assessment Body (CAB),
+* Audit Criteria, e.g. ETSI / WebTrust,
+* Name of Lead Auditor (except where prohibited by law, otherwise, we ask that you not provide any personally identifiable information),
+* For the Audit Team and the Audit Quality Reviewer, qualification information such as:
+    * Number of Audit Team Members,
+    * Academic qualifications or professional training received,
+    * Average Years of Auditing Experience auditing trust services or similar information systems,
+    * Experience, Special Skills, and Qualifications (e.g. audit/assessment principles and functions, information technology, software development, trust services, public key infrastructure, CA operations, and information security including risk assessment/management, network security, physical security, etc.), and
+    * Credentials, Designations, or Certifications (e.g. CPA, CISA, CITP, CISSP, CCSP/CCA/CCP, etc.).
+* How the Audit Team and team members are bound by law, regulation or professional standards to render an independent assessment of the CA (e.g.https://pub.aicpa.org/codeofconduct/Ethics.aspx# 0.300.050 Objectivity and Independence; CPA Canada, Rule 204; or Annex A of ETSI EN 319 403/403-1, respectively),
+* Name of the Insurance Carrier providing the professional liability or errors and omissions insurance coverage defined in CA/B Forum Baseline Requirements section 8.2., and
+* Whether the Audit Team relied on any third-party specialists or affiliate audit firms, and if so, their names and where they performed services.
+
+### 5.3 Notification of Cross-Certificate Issuance ###
+
+CAs must notify public@ccadb.org at least three weeks before issuing a new or replacement cross-certificate that can validate to a corresponding root CA certificate in one of the Stores. Individual Stores Program policies may require express approval before cross-certificate issuance. Corresponding Stores may choose to express cross-certificate issuance approval to the CA directly or through public@ccadb.org.
```

Comment:
This is a new requirement that the CCADB SC should discuss:
"CAs must notify public@ccadb.org at least three weeks before issuing a new or replacement cross-certificate that can validate to a corresponding root CA certificate in one of the Stores."

Reply:
"Individual Store program policies" sounds better.